src/estimators/tests/bloom_filters_test.py

```diff
@@ -157,7 +157,7 @@ def test_cardinality_estimation(self):
     b2 = BloomFilter(length=10, random_seed=3)
     b2.sketch[1] = 1
 
-    cardinality = UnionEstimator()([b1, b2])
+    cardinality = UnionEstimator()([b1, b2])[0]
```

Comment:
Like all the tests here, even though they may return the same result, I would say `sum(UnionEstimator()([b1, b2]))` would make more sense?

Reply:
Is this comment now moot?